src/cli/mod.rs

```diff
@@ -553,6 +572,12 @@ enum CreateWalletCommand {
         #[clap(short, long)]
         change_descriptor: String,
     },
+    SortedMultisig {
+        #[clap(short, long, num_args(2..) )]
```

Comment:
If you validate via `num_args` we may as well set the correct upper bound as well (15)